<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpinSlider</class>
 <widget class="QWidget" name="SpinSlider">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QToolButton" name="toolBtn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
